shell/platform/windows/client_wrapper/include/flutter/flutter_view_controller.h

```diff
@@ -62,28 +58,22 @@ class FlutterWindowController {
   FlutterDesktopPluginRegistrarRef GetRegistrarForPlugin(
       const std::string& plugin_name);
 
-  // The FlutterWindow managed by this controller, if any. Returns nullptr
+  // The FlutterView managed by this controller, if any. Returns nullptr
```

Comment:
This says it's a FlutterView, but is returning a raw pointer.

Reply:
Fixed by consolidation of view and controller